db/migrate/20160222113528_generate_tokens_for_existing_users.rb

```diff
@@ -0,0 +1,7 @@
+class GenerateTokensForExistingUsers < ActiveRecord::Migration
+  def change
+    User.all.each do |user|
```

Comment:
Its better to move data migration to custom rake task. Google how ti create custom rake task. The reason is that later on, when your project progresses - you can change the methods or class names used in this migration and your migrations won't run from start to end. Rails migrations should be used to update schema.